assets/js/broken_redirect_list.js

```diff
@@ -2570,6 +2571,7 @@ validurls['/docs/partners/message_orchestration/channel_extensions/ecommerce/sho
 validurls['/docs/partners/message_personalization/dynamic_content/contentful'] = '/docs/partners/message_orchestration/cms_dam/contentful';
 validurls['/docs/partners/message_orchestration/attribution/appsflyer/appsflyer'] = '/docs/partners/message_orchestration/deeplinking/appsflyer/appsflyer';
 validurls['/docs/partners/message_orchestration/attribution/branch/branch_for_deeplinking'] = '/docs/partners/message_orchestration/deeplinking/branch_for_deeplinking';
+validurls['/docs/partners/message_orchestration/attribution/branch/branch_for_attribution'] = '/docs/partners/message_orchestration/attribution/branch_for_attribution/';
```

Comment:
```suggestion
validurls['/docs/partners/message_orchestration/attribution/branch/branch_for_attribution'] = '/docs/partners/message_orchestration/attribution/branch_for_attribution/';
```
For some reason, this redirect isn't working in the preview 🤔

Reply:
My GOSH this one is stubborn. I could've sworn it worked in localhost. I'll try again lol

Reply:
@lydia-xie It worked in my localhost, so I'm not sure of the issue. Vercel is off for now, so I removed a `/` from the link, in case that changes it for you. Can you give another look?

Reply:
@rachel-feinberg it works! merging now 🫡